source/geometry_model/ellipsoidal_chunk.cc

```diff
@@ -53,11 +53,8 @@ namespace aspect
       semi_major_axis_a (-1),
       eccentricity (-1),
       semi_minor_axis_b (-1),
-      rot_para_to_para_angle (0),
-      para_to_rect_angle (0),
-      rotation_longitude (-1),
-      rotation_latitude (-1),
-      bottom_depth (-1)
+      bottom_depth (-1),
+      corners({{0}})

```

Comment:
You can't use this syntax, actually -- it requires support for C++11 which we don't assume yet everyone has.

Since `corners` is a `std::vector` and such objects have a default constructor that initializes it to be empty, my suggestion is to simply remove the entire line.